AMP-84095 add description of Braze User Alias Label (#964)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/destinations/braze.md

1. Select your **Braze API Endpoint**.
2. Enter your **Braze API Key**.
3. Enter your **Braze App ID**.

> 4. If you are using [Braze User Alias](https://www.braze.com/docs/api/objects_filters/user_alias_object) to identify your users on Braze, enter the **Braze User Alias Label** you plan on using. Otherwise this field can be safely ignored.

Reviewer: ```suggestion
4. If you use the [Braze User Alias](https://www.braze.com/docs/api/objects_filters/user_alias_object) to identify your users on Braze, enter the **Braze User Alias Label** you plan to use. Otherwise, you can ignore this field.
```

> 5. Check **Update Users Only** to only update users that exist in Braze, and not create new ones. If you're sending alias-only user profiles, don't check this field. If you check **Update Users Only**, your alias-only user profiles aren't created in Braze. More information available in the [Braze documentation](https://www.braze.com/docs/api/objects_filters/user_attributes_object).

Reviewer: ```suggestion
5. Check the **Update Users Only**  option to update users that exist in Braze, not create new users. If you send alias-only user profiles, don't check this field. If you check **Update Users Only**, your alias-only user profiles aren't created in Braze. More information available in the [Braze documentation](https://www.braze.com/docs/api/objects_filters/user_attributes_object).
```
